packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -693,9 +685,13 @@ const Summary: React.FunctionComponent = () => {
 
 				<div className="focused-launch-summary__ask-for-help">
 					<p>{ __( 'Questions? Our experts can assist.', __i18n_text_domain__ ) }</p>
-					<a href="/help" onClick={ onAskForHelpClick }>
+					<Button
+						isLink
+						href={ localizeUrl( 'https://wordpress.com/help', locale ) }
+						target="_blank"
```

Comment:
In the step-by-step-flow, the target is `_blank` too, hence I applied it here too.

Reply:
When we add `target="_blank"`, shouldn't we also add `rel="noopener noreferrer"`?

Reply:
Good call. I was assuming that `@wordpress/components` was taking care of this (partly because the linter didn't fire), but it doesn't (see [the component's code](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/button/index.js)). It might be a nice addition to the component but, for now, I've added it to our code and fixed it in the step-by-step launch as well.